### DISCUSSION TITLE
[request]: Show desktop notifications

### DISCUSSION THREAD

**Discussion by mr-cheffy** [community/community#57872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Woudn't it be cool if there was an option to allow user to recieve their notifications in their desktop enviroment instead of `https://github.com/notifications`?

**Comment by jamieomaguire** [comment]
I agree. I think this would literally save millions of engineer hours every day 👍 

It would make PR comment threads much easier to respond to.

It’s painfully typical for people to message you saying “I’ve replied to your PR comment by the way”. I think this could help prevent that. I don’t generally watch my email inbox that often for quick updates.

If you could opt-in, and manage the notification preferences to track specific things that would be incredibly useful.

**Reply by xander-marjoram** [reply]
So many hours are wasted either hoping that someone sees the automatic email (not a guarantee) or writing messages to people saying things like "I left some comments on your PR" or "I just replied to your comment".

**Comment by WoodyWoodsta** [comment]
Similar discussion from last year: https://github.com/orgs/community/discussions/9983
